esp/src/eclwatch/PackageMapPartsWidget.js

```diff
@@ -91,11 +91,15 @@ define([
              this.addPartsDropDown.on("apply", function (evt) {
                 if (context.addPartsDropDown.filterForm.validate()){
                     var addPartInput = context.getFilter();
+                    var packageMapSearch = context.params.packageMap.search("::");
+                    var packageMapClean;
+
+                    packageMapSearch > -1 ? packageMapClean = context.params.packageMap.split('::')[1] : packageMapClean = context.params.packageMap;
+
```

Comment:
I don't know why anyone would want to do it, but I was able to add a packagefile with :: in the name... which meant roxie::xxx::yyy.pkg was created, and there is also a way of setting the name explicitly to whatever you want.

The only way to support such names though is to specifically remove <target>:: from the beginning.  If the consensus is that this should be a known limitation so be it.